src/main/webapp/content/scss/course.scss

```diff
@@ -0,0 +1,18 @@
+.card-body {
```

Comment:
instead of a new global css file, i would use a component based style sheet

Reply:
Thanks for the feedback :)

Reply:
you're welcome. thanks for the improvement :)